packages/web-client/app/components/card-space/create-space-workflow/select-business-account/index.hbs

```diff
@@ -0,0 +1,92 @@
+<div {{did-insert this.setupBusinessAccountSelection}}>
+  {{#if (eq this.merchantSafes.length 0)}}
+    <ActionCardContainer
+      @header="Business account"
+      @isComplete={{@isComplete}}
+    >
+      <ActionCardContainer::Section>
+        <p>In order to create your Card Space, you must first create your first a business account. This is required
```

Comment:
Should this be a `WorkflowMessage` message instead? 🤔

Reply:
I imagined this working akin to how merchant creation [bounces you](https://github.com/cardstack/cardstack/blob/09351f488a9112357583191bcc7f0a23374e071b/packages/web-client/app/components/card-pay/create-merchant-workflow/index.ts#L224-L238) if you don’t have a prepaid card or prepaid card issuance [bounces you](https://github.com/cardstack/cardstack/blob/09351f488a9112357583191bcc7f0a23374e071b/packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/index.ts#L232-L241) if you don’t have a sufficient balance, maybe that pattern would work here too?

Reply:
Ah, I sensed there is a way where we already do similar things but couldn't remember where. Thanks!